src/utils/string.formatter.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * String formatting functions
+ */
+const REGEX_NON_PRINTABLE_CHARS: RegExp = /[^ -\xFF€]+/g;
+const REGEX_MULTIPLE_SPACES: RegExp = /\s\s+/g;
```

Comment:
Does this need to be a regex? would "  " not do?
Also what happens if there are 3 or more spaces?

Reply:
Yes we should use regex for this reason - for your second querstion have a look at the last unit test I added.